test/operator/installation.go

```diff
@@ -25,6 +31,40 @@ func InstallFromSource(withCsi bool) features.Func {
 	return manifests.InstallFromFiles(paths)
 }
 
+func InstallViaMake() features.Func {
+	return func(ctx context.Context, t *testing.T, environmentConfig *envconf.Config) context.Context {
+		rootDir := getCorrectDir()
+
+		platform := kubeobjects.ResolvePlatformFromEnv()
+		makeTarget := "deploy"
+		if platform == kubeobjects.Openshift {
+			makeTarget = strings.Join([]string{makeTarget, "openshift"}, "/")
+		} else if platform == kubeobjects.Kubernetes {
+			makeTarget = strings.Join([]string{makeTarget, "kubernetes"}, "/")
+		}
```

Comment:
Add some handling of the `else` branch here. Otherwise you'll end up running `make deploy`, which doesn't work (`make: *** No rule to make target 'deploy'.  Stop.`) It's always better to prevent an error early than catch it later.